doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -1,15 +1,230 @@
 .. _readme-buildingdocs:
 
+.. _Chapel Developers: https://chapel.discourse.group/c/developers
+
 =============================
 Building Chapel Documentation
 =============================
+The live Chapel web documentation lives at https://chapel-lang.org/docs/.
+All of the source for the documentation is in the chapel source repository, and
+this note talks about how those are organized and built into the html documentation
+seen on the website.
 
 The easiest way to build the Chapel web documentation is to use the top level
-make target::
+of your chapel github clone
 
+make target::
    [g]make docs
```

Comment:
This is a little confusing to me. Would a better way to word this be something like:
"The easiest way to build the Chapel web documentation is by running this command make target::[g]make docs from the top level of your local Chapel repository"?
The message as is seems to be a mashing of 2 different things, the make target and someone's local repository. Additionally (and this is seen a few other places in this doc as well), there is a bit of inconsistency in the capitalization. On line 13, Chapel is capitalized; on line 14, neither Chapel or GitHub are capitalized. This is, of course, not a big deal, but I always notice those minor inconsistencies, so I am likely just nitpicking!

Reply:
I cleaned up that sentence about making the documentation.  I put this instance of chapel in backquotes because I am referring to the actual clone directory name, which is lowercase.